src/main/java/io/temporal/samples/updatabletimer/UpdatableTimer.java

```diff
@@ -35,8 +35,8 @@
 
   public void sleepUntil(long wakeUpTime) {
     Instant wakeUpInstant = Instant.ofEpochMilli(wakeUpTime);
-    LocalDateTime date = wakeUpInstant.atZone(ZoneId.systemDefault()).toLocalDateTime();
-    logger.info("sleepUntil: " + date);
+    LocalDateTime date = wakeUpInstant.atZone(ZoneId.of("UTC")).toLocalDateTime();
```

Comment:
Should use ZoneOffset.UTC instead

Reply:
also can you please add comment on why this was changed? to explain users. 
raises a question, since date is just used for logging purposes, does not not make more sense to display it in users local timezon? Who knows UTC time really?

Reply:
Thanks, @tsurdilo. Good point, I simplified the time conversion. `Instant`, indeed, uses UTC and also includes the `Z` suffix into the `toString` value.

Reply:
Could you please suggest more specifically what you would like to be in the comment?

You are right that in this particular instance the "side effect" does not matter, but it seems to me that in samples/tutorials, we should be particularly consistent with avoiding non-deterministic APIs, just to set a good example for the users.

In general, to make the log most readable one would want to use the local time of the _reader_, not of the _producer_. Of course, that is not as easy in this case. But using the UTC time seems like a good practice anyway.

All this is mondain, though, so not sure if this is worth mentioning in comments. Please let me know if you disagree. :)